dashboard/app/models/programming_expression.rb

```diff
@@ -66,8 +66,7 @@ def self.properties_from_file(path, content)
 
     environment_name = File.basename(File.dirname(path)) == 'GamelabJr' ? 'spritelab' : File.basename(File.dirname(path))
     programming_environment = ProgrammingEnvironment.find_by(name: environment_name)
-
-    syntax = ProgrammingExpression.get_syntax(expression_config)
+    throw "Cannot find ProgrammingEnvironment #{environment_name}" unless programming_environment
```

Comment:
nit: can just use `find_by!` on the previous line if you want this behavior, unless you prefer this error message (I forget what the default one includes)

Reply:
When I try this on the command line, I get
```
[development] dashboard > ProgrammingEnvironment.find_by!(name: 'blah')
  ProgrammingEnvironment Load (6.6ms)  SELECT  `programming_environments`.* FROM `programming_environments` WHERE `programming_environments`.`name` = 'blah' LIMIT 1
Traceback (most recent call last):
        1: from (irb):7
ActiveRecord::RecordNotFound (Couldn't find ProgrammingEnvironment)
```

I'd like to have the name of the missing environment in this case, so I'd rather keep it as is

Reply:
👍  makes sense!